IronSoftware.Drawing/IronSoftware.Drawing.Common.Tests/UnitTests/AnyBitmapFunctionality.cs

```diff
@@ -132,6 +132,35 @@ public void Try_Save_Bitmap_with_Format()
             CleanResultFile("result-try-png-loss.png");
         }
 
+        [FactWithAutomaticDisplayName]
+        public void Check_Bitmap_Colors()
+        {
+            string imagePath = GetRelativeFilePath("colors.png");
+            var anyBitmap = new AnyBitmap(imagePath);
```

Comment:
Chiming in. If i'm not wrong on this one, we shall `using var anyBitmap = new ...`? and this applies to other occurrences as well.

Reply:
Oh yes, it should be disposed properly. Will fix.

Reply:
I think in some tests it makes more sense to Dispose at the end, so will update those as well.

However, since it's tests, I think it's not a huge concern. Probably gets disposed automatically when test completes.

Reply:
Sure agree Wolf. Anyhow, prevention from it to resurface later would be great. Because there is the same incidence related to `IDispose` in IronOcr that would fail unexpectedly, after we properly destroy the memory, it passes.

I'm not 100% how xunit runs all the individual tests. But I believe it runs all the tests within a single process. So if we miss destroying the memory in one of the unit tests, such leaked memory will carry out until all tests would be done. Because all of them share the same memory region. It potentially has a chance to fail another unrelated unit test due to no more memory left to carry out such test.

Reply:
I also think it's just general good practice to always Dispose anything that needs to be disposed (implements `IDisposable`). Reason: if someone copies and pastes code, they will be aware as well.

I just copied my code from another unit test and made some changes, hence I didn't really consider this memory issue.